src/types.ts

```diff
@@ -9,34 +9,58 @@ export enum TransactionAddressType {
   Home = 'home'
 }
 
-export type RegisterSignupProps = {
+export type RegisterSignupBaseProps = {
+  accountId?: string
+  policyId?: string
+  [x: string]: any
+}
+
+export type RegisterSignupProps = RegisterSignupBaseProps & {
   installationId: string
   addressCoordinates?: AddressCoordinates
   addressLine?: string
   structuredAddress?: StructuredAddress
-  accountId?: string
   externalId?: string
-  policyId?: string
-  [x: string]: any
 }
 
-export type SignupResponse = {
+export type RegisterWebSignupProps = RegisterSignupBaseProps & {
+  sessionToken: string
+}
+
+export type SignupBaseResponse = {
   id: string
-  requestId: string
-  deviceId: string
+  installationId: string
   riskAssessment: RiskAssessment
   reasons: Array<Reason>
+}
+
+export type SignupResponse = SignupBaseResponse & {
+  requestId: string
```

Comment:
This should be on `SignupBaseResponse`. Web signups also have the `requestId` in the response.